test/Directory.Build.targets

```
@@ -28,7 +28,9 @@
     <ProjectReference Include="$(RepoRoot)src\Platform\Microsoft.Testing.Extensions.TrxReport\Microsoft.Testing.Extensions.TrxReport.csproj" />
     <PackageReference Include="Microsoft.Testing.Extensions.CodeCoverage" GeneratePathProperty="True" />
     <PackageReference Include="MSTest.Analyzers" />
+
     <PackageReference Include="MSTest.TestFramework" />
+    <PackageReference Update="MSTest.TestFramework" Condition=" '$(UseMSTestFromSource)' == 'true' " ExcludeAssets="runtime;compile" />
```

I guess that works too, I was envisioning something like


```suggestion
     <PackageReference Include="MSTest.TestFramework" Condition=" '$(UseMSTestFromSource)' != 'true' " />
```

---

I'll merge as-is to unblock ci. Feel free to do a follow-up if you find my syntax better.